Tests/UnitTests/Core/Vertexing/LinearizedTrackFactoryTests.cpp

```diff
@@ -77,12 +79,16 @@ std::uniform_real_distribution<> phiDist(-M_PI, M_PI);
 std::uniform_real_distribution<> thetaDist(1.0, M_PI - 1.0);
 // Track charge helper distribution
 std::uniform_real_distribution<> qDist(-1, 1);
+// Track time distribution
+std::uniform_real_distribution<> tDist(-0.002_ns, 0.002_ns);
 // Track IP resolution distribution
 std::uniform_real_distribution<> resIPDist(0., 100_um);
 // Track angular distribution
 std::uniform_real_distribution<> resAngDist(0., 0.1);
 // Track q/p resolution distribution
 std::uniform_real_distribution<> resQoPDist(-0.1, 0.1);
+// Track time resolution distribution
+std::uniform_real_distribution<> resTDist(0.1_ns, 0.2_ns);
```

Comment:
```suggestion
std::uniform_real_distribution<> resTDist(-0.1_ns, 0.1_ns);
```
time residual should be symmetric around 0 I think? not that it makes a big difference but looks confusing to me

Reply:
This is the variance of the time, so I guess it makes sense that it is non-symmetric (see further down in the code). It is however confusing that resQoPDist is symmetric even though we only use squared values from it. I changed this!

Reply:
from the name I would suspect that it is the residual of the parameter and then we square it to get something like the variance

Reply:
Oh I understand now, you are right. I will fix this!